src/js/tracing/nativeframes.ts

```diff
@@ -0,0 +1,263 @@
+import { Span, Transaction } from "@sentry/tracing";
+import { Event, EventProcessor, Measurements } from "@sentry/types";
+import { logger, timestampInSeconds } from "@sentry/utils";
+
+import { NativeFramesResponse } from "../definitions";
+import { NATIVE } from "../wrapper";
+import { instrumentChildSpanFinish } from "./utils";
+
+interface FramesMeasurements extends Measurements {
+  frames_total: {
+    value: number;
+  };
+  frames_slow: {
+    value: number;
+  };
+  frames_frozen: {
+    value: number;
+  };
+}
```

Comment:
```suggestion
type FramesMeasurements = Record<'frames_total' | 'frames_slow' | 'frames_frozen', { value: number }>;
```

let's also not extend from the `Measurements` type, it allows for arbitrary keys which we don't want here.